programs/clearing_house/src/math/bank_balance.rs

```diff
@@ -259,3 +287,56 @@ pub fn check_withdraw_limits(bank: &Bank) -> ClearingHouseResult<bool> {
 
     Ok(valid_withdrawal)
 }
+
+pub fn validate_bank_balances(bank: &Bank) -> ClearingHouseResult<u64> {
```

Comment:
did a similar check at the end of withdraw, you might wanna put this there now and get rid of my check